sdk/ml/azure-ai-ml/azure/ai/ml/entities/_compute/compute_instance.py

```diff
@@ -230,12 +242,22 @@ def _to_rest_object(self) -> ComputeResource:
                 )
             )
 
+        idle_time_before_shutdown = None
+        if self.idle_time_before_shutdown_minutes:
+            idle_time_before_shutdown = f"PT{self.idle_time_before_shutdown_minutes}M"
+        elif self.idle_time_before_shutdown:
+            warnings.warn(
+                "This property is deprecated",
```

Comment:
I think we should keep your original message:
```suggestion
                "This property is deprecated. Please use 'idle_time_before_shutdown_minutes' instead.",
```

Reply:
Thanks for adding this suggestion!